docs/pages/access-controls/sso/oidc.mdx

```diff
@@ -193,6 +193,29 @@ spec:
        roles: [ "access" ]
 ```
 
+### Optional: Max age
+
+Teleport has supported setting the `max_age` field since version 13.3.7 to control the
```

Comment:
Not a big deal, but in my original suggestion I recommended making this note in the v13 backport only.

It doesn't really need to be in master or v14 since _all_ of those versions support max_age.

Reply:
Ah I must have missed that, makes sense. Should I make a follow-up PR and backport it to v14 to remove the version note you think?

Reply:
If you're feeling especially motivated you are welcome to, but if I were you I probably wouldn't worry about it.

Reply:
Cool I'll just leave it as it is then thanks